APISpecification.yaml

```diff
@@ -381,6 +441,21 @@ components:
             name:
               type: string
               example: GRCh38.p13
+        release:
+          type: object
+          properties:
+            release_date:
+              type: string
+              example: 2023-10-18
+            release_type:
+              type: string
+              example: partial
+            release_name:
+              type: string
+              example: 2023-10-18
+            is_current:
+              type: boolean
+              example: true
```

Comment:
If the shape of release here is the same as the shape of release coming out of the `/releases` endpoint, then shouldn't line 444 just reference the release object on line 1076?

Reply:
> If the shape of release here is the same as the shape of release coming out of the /releases endpoint, then shouldn't line 444 just reference the release object on line 1076?

Yes, you're right, thank you for pointing that out, I've done the changes

> I am confused about the meaning of the is_current field here.

It's the one attached to a release, would you like me to call it somthing more explicit like `release_is_current`?